<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tengu</class>
 <widget class="QMainWindow" name="Tengu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>218</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Tengu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frameMain">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <widget class="QFrame" name="topFrame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>63</width>
         <height>74</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="ButtonWallet">
          <property name="text">
           <string>钱包</string>
          </property>
          <property name="icon">
           <iconset resource="res/Images.qrc">
            <normaloff>:/Buttons/wallet.svg</normaloff>:/Buttons/wallet.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QToolButton" name="ButtonClose">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>31</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>✖️</string>
       </property>
      </widget>
      <widget class="QToolButton" name="ButtonMin">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>122</y>
         <width>36</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>➖</string>
       </property>
      </widget>
      <widget class="QToolButton" name="ButtonMax">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>94</y>
         <width>36</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>➕</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action1"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="action1">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res/Images.qrc"/>
 </resources>
 <connections/>
</ui>
